resources/js/requests/components/RequestsListing.vue

```diff
@@ -26,177 +26,197 @@
-                return data;
-            },
-            fetch() {
-                this.loading = true;
-                this.additionalParams = this.additionalParams ? this.additionalParams : '&include=assigned';
-
-                // Load from our api client
-                ProcessMaker.apiClient
-                    .get(
-                        "requests?page=" +
-                        this.page +
-                        "&per_page=" +
-                        this.perPage +
-                        "&include=process,delegations,delegations.user" +
-                        "&filter=" +
-                        this.filter +
-                        "&order_by=" +
-                        (this.orderBy === '__slot:ids' ? 'id' : this.orderBy) +
-                        "&order_direction=" +
-                        this.orderDirection +
-                        this.additionalParams
-                    )
-                    .then(response => {
-                        this.data = this.transform(response.data);
-                        this.loading = false;
-                    });
-            }
+        {
+          title: "Process",
+          name: "name",
+          sortField: "name"
+        },
+        {
+          title: "Status",
+          name: "status",
+          sortField: "status"
+        },
+        {
+          title: "Stage",
+          name: "stage"
+        },
+        {
+          title: "Participants",
+          name: "assigned",
+          callback: this.assignedTo
+        },
+        {
+          title: "Started",
+          name: "created_at",
+          sortField: "created_at"
+        },
+        {
+          title: "Completed",
+          name: "completed_at",
+          sortField: "completed_at"
+        },
+        {
+          title: "Duration",
+          name: "duration_at"
+        },
+        {
+          name: "__slot:actions",
+          title: ""
         }
+      ]
     };
+  },
+  methods: {
+    openRequest(data, index) {
+      window.open("/requests/" + data.id, "_self");
+    },
+    assignedTo(delegations) {
+      let assignedTo = "";
+      if (!delegations) return assignedTo;
+      let count = 0;
+      let usedAvatar = [];
+      delegations.forEach(function(delegation, key) {
+        if (
+          delegation.user &&
+          usedAvatar.includes(delegation.user.id) === false
+        ) {
+          usedAvatar.push(delegation.user.id);
+
+          if (key <= 4) {
+            let user = delegation.user;
+            assignedTo += user.avatar
+              ? '<img class="avatar-image-list avatar-circle-list" src="' +
+                user.avatar +
+                '" title="' +
+                user.fullname +
+                '"> '
+              : '<button type="button" class="avatar-circle-list" title="' +
+                user.fullname +
+                '">' +
+                '<span class="avatar-initials-list">' +
+                user.firstname.charAt(0).toUpperCase() +
```

Comment:
charAt will have problems with unicode characters.

Reply:
Corrected accessing the unicode array from the string